packages/handlersjs-http/lib/handlers/http-handler-static-asset.spec.ts

```diff
@@ -69,12 +70,25 @@ describe('HttpHandlerStaticAssetService', () => {
 
     });
 
-    // it('should return file content when file is found', async() => {
+    // it('should return file content when file is found with a relative path', async() => {
+
     //   context.request.parameters.filename = 'test.txt';
     //   const response = await service.handle(context).toPromise();
     //   expect(response.body).toBe('test file');
+
     // });
```

Comment:
This test was already commented. I tried to uncomment it, but this does not work in commit hook (we need a strategy to load test files from everywhere...)

Reply:
Yeah this is the test I 'fixed' in #105  by mocking the readFile implementation. Not really the way to go, I know.